plugins/entando-plugin-jpdigital-exchange/src/main/resources/component/plugins/digitalExchange/commands.xml

```diff
@@ -6,21 +6,40 @@
             <value>@pageModelController.addPageModel(@modelParser.parse('%s', T(org.entando.entando.web.pagemodel.model.PageModelRequest)), #bindingResult())</value>
         </entry>
         <entry>
+            <key>deletePageModel</key>
+            <value>@pageModelController.deletePageModel('%s')</value>
+        </entry><entry>
             <key>createWidget</key>
             <value>@widgetController.addWidget(@modelParser.parse('%s', T(org.entando.entando.web.widget.model.WidgetRequest)), #bindingResult())</value>
         </entry>
+        <entry>
+            <key>deleteWidget</key>
+            <value>@widgetController.deleteWidget('%s')</value>
+        </entry>
         <entry>
             <key>createGroup</key>
             <value>@groupController.addGroup(@modelParser.parse('%s', T(org.entando.entando.web.group.model.GroupRequest)), #bindingResult())</value>
         </entry>
+        <entry>
+            <key>deleteGroup</key>
+            <value>@groupController.deleteGroup('%s')</value>
+        </entry>
         <entry>
             <key>createRole</key>
             <value>@roleController.addRole(@modelParser.parse('%s', T(org.entando.entando.web.role.model.RoleRequest)), #bindingResult())</value>
         </entry>
+        <entry>
+            <key>deleteRole</key>
+            <value>@roleController.deleteRole('%s')</value>
+        </entry>
         <entry>
             <key>createLabel</key>
             <value>@labelController.addLabelGroup(@modelParser.parse('%s', T(org.entando.entando.web.label.LabelRequest)))</value>
         </entry>
+        <entry>
+            <key>deleteLabel</key>
+            <value>@labelController.deleteLabelGroup('%s')</value>
+        </entry>
         <entry>
             <key>createContentType</key>
             <value>@contentTypeResourceController.create(@modelParser.parse('%s', T(com.agiletec.plugins.jacms.aps.system.services.contentmodel.model.ContentTypeDtoRequest)), #bindingResult())</value>
```

Comment:
Please add also entries for deleting content types and content models, even they are only manually testable